AdobeStockImageAdminUi/Test/Mftf/Test/AdminAdobeStockLocalizationTest.xml

```diff
@@ -32,11 +32,13 @@
             <actionGroup ref="logout" stepKey="logout"/>
         </after>
         <!-- verify that user can search adobe stock image in ru_RU locale -->
-        <actionGroup ref="AdminAdobeStockOpenSearchModalActionGroup" stepKey="openSearchModal"/>
+        <actionGroup ref="AdminOpenMediaGalleryForPageNoEditorActionGroup" stepKey="openMediaGalleryForPage"/>
+        <actionGroup ref="AdminAdobeStockOpenPanelFromMediaGalleryActionGroup" stepKey="openAdobeStockPanel"/>
         <actionGroup ref="AdminSearchImagesOnModalActionGroup" stepKey="searchForPopular">
             <argument name="query" value="Автомобили"/>
         </actionGroup>
         <actionGroup ref="AdminAdobeStockExpandImagePreviewActionGroup" stepKey="expandImagePreview"/>
+        <!-- TODO: the following assertion/requirement on the specific keyword we expect wont work as we have no guarantee on the exact keyword to use as the image could be random -->
```

Comment:
Hi @filmaj. My understanding is that you may not get the same image every time, right? But would it be the case even though you're searching for a specific keyword?  Maybe searching for an image ID stored in `.ENV` would help getting the same image every time? Maybe I'm understanding your comment wrong. Thanks!

Reply:
Hey @drpayyne, yes that  is correct. At least, that  was my experience when running the test locally - the images that came up with the search term would have various keywords and  did not match the keyword expectation in this test.

Maybe there is a better method of honouring the spirit of this test with a different approach. Ultimately what we want to achieve is that we want to validate that we receive results in the correct language/locale. Is there a different way we could go about it?

Maybe this is worth spinning out into its own issue?